src/main/scala/centaur/test/Test.scala

```diff
@@ -80,17 +80,22 @@ object Operations {
     * stops with a failure.
     */
   def pollUntilStatus(workflow: SubmittedWorkflow, expectedStatus: WorkflowStatus): Test[SubmittedWorkflow] = {
+    def pollDelay() = Thread.sleep(10000) // This could be a lot smarter, including cromwell style backoff
```

Comment:
No. I won't even pretend like the number 2 had any sort of logical basis. At least I know where the 10 seconds came from and that was purely using the other one. From what I saw it was usually < 1s TBH